posts/20240229/index.qmd

```diff
@@ -0,0 +1,71 @@
+3. The authors show that one can use this logic of nonlinear/discontinuous metrics to 'induce' emergence in other scenarios, notably vision tasks. This induced behavior is summarized in the figures below.
+
+![](visiontasks.png)
+
+## Our Discussion
+
+We found this paper enjoyable and illuminating. The paper had a relatively simple main message, and argued their point compellingly, although we did have some confusion about the details of certain analyses. In particular, the meta-analysis of claimed emergent ability using BIG-Bench (Section 4) raised several questions. For example, we were not sure how the authors went from a distribution of emergence scores to a cutoff for 'possible emergent abilities'.  The figure for this section seemed to come from two sources; one visualized the purely quantitative emergence scores from BIG-Bench, while the other depicted hand-annotated data from elsewhere. It was not well explained how these were linked. Additionally, accuracy was previously described as an emergent-prone metric, but that metric showed no suggestion of emergence in this analysis. This was not discussed.
+
+As the authors mention in their discussion, this line of work does not rebut the possibility of emergence nor does it really dictate which metrics are best for analyzing a single model. Rather, it simply suggests that if your goal is to model performance across changing model scores, attention should be paid to use metrics which are linear or continuous with the per-token correctness probability.
+
+As a final note, we discussed how ML papers often have much stronger and eye-catching language than many other academic disciplines. In some cases, this can be a good thing to attract attention, but it is also easy to misinterpret and make stronger. For example, the aforementioned [Vice article](https://www.vice.com/en/article/wxjdg5/scary-emergent-ai-abilities-are-just-a-mirage-produced-by-researchers-stanford-study-says) attaches the qualifier 'scary' to its headline when mentioning the theorized emergent AI capabilities. This kind of language makes it easy for the conversation to shift away from the core of the article and into more vague notions of 'sentience' and AI existential risk. These latter topics are certainly important and interesting, but they are mostly divorced from the actual content of this particular paper.
```

Comment:
The final note pretty much sums up the main motivation of the paper (the usage of the term "emergence") and addresses the widespread issue of flowery language. And thanks for the vice article mention!